docs/machine-learning/resources/learners.md

```diff
@@ -0,0 +1,67 @@
+# Learning Algorithms
+The following table will help you understand the different learning algorithms (learners) included in ML.NET and the [machine learning tasks](tasks.md) that they can be used for. Click on the learner name for more in-depth explanations on each of these learning methods.
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all learning algorithms are currently supported. To submit a request for a certain learner, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
+
+| Learner Name | Binary classification | Multi-class classification | Regression | Ranking | Clustering | Anomaly Detection |
+| --- |:---:|:---:|:---:|:---:|:---:|:---:|
+| [Averaged Perceptron](#averaged-perceptron) |:heavy_check_mark:||||||
+| [FastForest](#fastforest) |:heavy_check_mark:||:heavy_check_mark:||||
+| [FastTree](#fasttree)|:heavy_check_mark:||:heavy_check_mark:|:heavy_check_mark:|||
+| [Generalized Additive Models](#generalized-additive-models) |:heavy_check_mark:||:heavy_check_mark:||||
+| [K-Means++](#k-means) |||||:heavy_check_mark:||
+| [Linear SVM](#linear-svm) |:heavy_check_mark:||||||
+| [Logistic Regressor](#logistic-regressor) |:heavy_check_mark:|:heavy_check_mark:|||||
+| [Naïve Bayes](#naïve-bayes) ||:heavy_check_mark:|||||
+| [Online Gradient Descent](#online-gradient-descent) |||:heavy_check_mark:||||
+| [Ordinary Least Squares](#ordinary-least-squares) |||:heavy_check_mark:||||
+| [PCA Anomaly Detector](#pca-anomaly-detector) ||||||:heavy_check_mark:
+| [Poisson Regression](#poisson-regression) |||:heavy_check_mark:||||
+| [Stochastic Dual Coordinate Ascent](#stochastic-dual-coordinate-ascent) |:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:||||
+| [Stochastic Gradient Descent](#stochastic-gradient-descent) |:heavy_check_mark:||||||
+| [Tweedie FastTree](#tweedie-fasttree) |||:heavy_check_mark:||||
+
+## Averaged Perceptron
+
+
+## FastForest
+
+
+## FastTree
+
+
+## Generalized Additive Models
+
+
+## K-Means++
+
+
+## Linear SVM
+
+
+## Logistic Regressor
```

Comment:
Each learner has description in the [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.logisticregressionbinaryclassifier?view=ml-dotnet) based on the description in the [code](https://github.com/dotnet/machinelearning/blob/1b08ae43887887abb478130e070dd8407c5e17b9/src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs#L389). I'd recommend avoiding duplication if possible, linking to the other description, and improving the description in the code.